lib/active_model/serializable_resource.rb

```diff
@@ -74,6 +75,14 @@ def serializer?
       use_adapter? && !!(serializer)
     end
 
+    def instrument
```

Comment:
would a logger be a useful place to integrate with active support notifications? https://github.com/rails-api/active_model_serializers/pull/1089